src/schedule.rs

```diff
@@ -135,9 +135,17 @@ impl Schedule {
         let seconds = Seconds::from_field(iter.next().unwrap())?;
         let minutes = Minutes::from_field(iter.next().unwrap())?;
         let hours = Hours::from_field(iter.next().unwrap())?;
-        let days_of_month = DaysOfMonth::from_field(iter.next().unwrap())?;
+        let days_of_month_field = iter.next().unwrap();
+        let days_of_month_is_any = days_of_month_field.is_any();
+        let days_of_month = DaysOfMonth::from_field(days_of_month_field)?;
         let months = Months::from_field(iter.next().unwrap())?;
-        let days_of_week = DaysOfWeek::from_field(iter.next().unwrap())?;
+        let days_of_week_field = iter.next().unwrap();
+        if days_of_month_is_any && days_of_week_field.is_any() {
```

Comment:
Could be useful to note somewhere that `"* * * * * *"` is [still] supported along with `"* * * ? * *"`, but not `""* * * ? * ?"`.

Reply:
I pushed a commit with this added as a comment to the top of this function, but if you had something different in mind for either the contents or location just let me know.